fiduswriter/base/static/js/modules/app/index.js

```diff
@@ -1,6 +1,5 @@
-import OfflinePluginRuntime from 'offline-plugin/runtime'
 import {CSL} from 'citeproc-plus'
-
+import OfflinePluginRuntime from 'offline-plugin/runtime' //eslint-disable-line
```

Comment:
@anandksalien Why do we need/want to import OfflinePluginRuntime if we then don't use it anywhere in the file?

Reply:
Hey @johanneswilm . I checked this out. We need the import, only then the sw.js is generated and put into static-transpile folder. If this is not imported the sw.js is not generated. Please refer this link for more details [https://goo.gl/YwewYp]( https://goo.gl/YwewYp)

Reply:
Hey @anandksalien on that link they also call the install method. But in our case we remove the install call.

Reply:
@johanneswilm Yeah I noticed it. But for some reason when I try and remove that import it throws this error while collecting static and doesn't generate the sw.js that is needed.
![image](https://user-images.githubusercontent.com/46891412/76620949-566f1480-6554-11ea-8a8b-69d2279090a9.png)
This was the reason why i kept it for now.Don't know if it's really an issue with the library.